src/components/YourCompany/OfficeAddresses.vue

```diff
@@ -37,27 +37,31 @@
           <div v-else>Same as Mailing Address</div>
         </v-flex>
 
-        <v-flex xs1 class="mt-n2" v-if="officeAddressesChanged">
+        <v-flex xs1 v-if="officeAddressesChanged">
           <div class="actions mr-4">
             <span class="edit-action">
               <v-btn
                 text color="primary"
                 id="btn-undo-office-addresses"
-                @click="resetOfficeAddresses()"
+                @click="resetOfficeAddresses(); dropdown=false"
```

Comment:
Didn't know you could double up on these events. Cool

Reply:
Yup, it's just JS.

Reply:
I will add that any code in the template should have to do directly with the template (not business logic).